services/common/src/components/project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,472 @@
+              </Col>
+            </Row>
+          </Col>
+          <Col span={24}>
+            <Typography.Paragraph>
+              Below are the authorization submissions and their status in the project description.
+              Both the Major Mines Office and Ministry of Environments reviews must be completed for
+              this stage to be considered complete.
+            </Typography.Paragraph>
+
+            <Callout
+              message={
+                <div className="nod-callout">
+                  <h4>Submission Failed</h4>
+                  <p>
+                    One or more of your environment authorization applications has not been
+                    submitted successfully. Please retry the submission.
+                  </p>
+                </div>
+              }
+              severity={CALLOUT_SEVERITY.danger}
+            />
+
+            <Typography.Title level={3}>Submission Progress</Typography.Title>
+            <Typography.Title level={4}>Major Mines Office</Typography.Title>
+            <Typography.Title level={5}>Mines Act</Typography.Title>
+            <CoreTable dataSource={minesActData} columns={minesActColumns} />
+            <Typography.Title level={4}>Ministry of Environment</Typography.Title>
+            <Typography.Title level={5}>Environmental Management Act</Typography.Title>
+            {environmentalManagementActData?.length > 0 && (
+              <Alert
+                message="Submission Failed Please Retry"
+                showIcon
+                type="error"
+                description={`Your environment authorization application was not submitted successfully. Please retry the submission or start a new application for the rejected authorization(s). You can link the submission to the new application on the Related Projects page. One or more of your environment authorization application has not been submitted successfully. Please retry the submission.`}
+                action={
+                  shouldDisplayRetryButton ? (
+                    <Button onClick={handleRetryAMSSubmissionClicked}>
+                      Retry Failed Submission
+                    </Button>
+                  ) : null
+                }
+                style={{ marginBottom: "12px" }}
+              />
+            )}
+            <CoreTable
+              rowKey="project_summary_authorization_guid"
+              dataSource={environmentalManagementActData}
+              columns={otherActColumns}
+            />
+            <br />
+            <Typography.Title level={5}>Water Sustainability Act</Typography.Title>
+            <CoreTable
+              rowKey="project_summary_authorization_guid"
+              dataSource={waterSustainabilityActData}
+              columns={otherActColumns}
+            />
+            <br />
+            <Typography.Title level={5}>Forestry Act</Typography.Title>
+            <CoreTable dataSource={forestryActData} columns={otherActColumns} />
```

Comment:
just this instance of the table here is missing the rowKey. 😅 
I'm assuming it's also `project_summary_authorization_guid`

Reply:
the rowKey has been added now